static/js/index.js

```diff
@@ -138,6 +143,31 @@ function getInitialDate() {
   }
   return date;
 }
+function dateToNextTri() {
+  // TRI_START_ARR stores all the start of tri dates
+  let goToTri = -1; // stores index of trimester you want to go to
+  // default to -1 if no tri is greater than current date
+  for (let currLooking = 0; currLooking < 3; currLooking++){
+    if (globalDate < TRI_START_ARR[currLooking]){
+      // located the tri to go to
+      goToTri = currLooking; // set it and break
+      break;
+    }
+  }
+  if (goToTri == -1){
+    // if it is greater than the start of the spring tri
+    // then loop back to the fall tri
+    goToTri = 0;
```

Comment:
do we need a way to make it jump back to today's date?

Reply:
I feel like we don't really need that, seeing as they could reload the page and most of the time there isn't a need to go back to today